### PR TITLE
[4.1] Lazy Scheduler disabled

### PR DESCRIPTION
**Pull request by brianteeman**
Follow the advice of @philetaylor to amend the language string

Pull Request for Issue #36659

### CODE REVIEW

File: administrator/language/en-GB/com_scheduler.ini [hunk 1]
```diff
@@ -12,7 +12,7 @@ COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_KEY_LABEL="Global Key"
 COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_LINK_DESC="By default, requesting this base link will only run tasks due for execution. To execute a specific task, use the task's ID as a query parameter appended to the URL: <code>BASE_URL&id=&lt;taskId&gt;</code>"
 COM_SCHEDULER_CONFIG_GLOBAL_WEBCRON_LINK_LABEL="Webcron Link (Base)"
 COM_SCHEDULER_CONFIG_HASH_PROTECTION_DESC="If enabled, tasks will only be triggered when URLs have the scheduler hash as a parameter."
-COM_SCHEDULER_CONFIG_LAZY_SCHEDULER_ENABLED_DESC="If disabled, scheduled tasks will not be triggered by visitors on the site.<br>Recommended if triggering with native cron."
+COM_SCHEDULER_CONFIG_LAZY_SCHEDULER_ENABLED_DESC="If disabled, scheduled tasks will not be triggered by visitors on the site.<br>It is recommended TO DISABLE the Lazy Scheduler, if triggering with native cron."
```
Comment (chmst) [inline administrator/language/en-GB/com_scheduler.ini]: suggestion:  .. It is recommended to DISABLE ....

Comment (PhilETaylor) [reply administrator/language/en-GB/com_scheduler.ini]: That is literally what it says!

Comment (chmst) [reply administrator/language/en-GB/com_scheduler.ini]: my suggestion is writing the word TO in lower case - maybe a  non-native speaker thing

**Comment by chmst** [comment]
I have tested this item :white_check_mark: successfully on cf00f64b203da13cf4cce17efbffa4133a8d6802<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37033">issues.joomla.org/tracker/joomla-cms/37033</a>.</sub>

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on cf00f64b203da13cf4cce17efbffa4133a8d6802<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37033">issues.joomla.org/tracker/joomla-cms/37033</a>.</sub>

**Comment by Quy** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/37033">issues.joomla.org/tracker/joomla-cms/37033</a>.</sub>

**Comment by bembelimen** [comment]
Thx

**Comment by brianteeman** [comment]
thx

**Pull request merged**